scripts/transpilation/transpiled-build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+#
+# Backpack - Skyscanner's Design System
+#
+# Copyright 2016 Skyscanner Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+lerna clean --yes
+
+rm -rf ./dist
+mkdir "dist"
+cd ./packages
+
+for d in */
+do
+  cd "$d" || return
+  if [ "$d" == "bpk-component-barchart/" ]
+  then
+      cp -R "../../node_modules/d3-array" "./d3-array"
+      cp -R "../../node_modules/d3-color" "./d3-color"
+      cp -R "../../node_modules/d3-format" "./d3-format"
+      cp -R "../../node_modules/d3-interpolate" "./d3-interpolate"
+      cp -R "../../node_modules/d3-scale" "./d3-scale"
+      cp -R "../../node_modules/d3-time" "./d3-time"
+      cp -R "../../node_modules/d3-time-format" "./d3-time-format"
+      cp -R "../../node_modules/internmap" "./internmap"
+  fi
```

Comment:
what's the reason. behind doing something custom here for d3? could you add a comment explaining why?

Reply:
It allows us to transpile those deps. I faced issues with D3 minified build imports, so this script transpiles d3 modules and includes them into bundle. Not sure if it's the best way to solve this issue, open to discussion

Reply:
ok, happy for now how we have it, we just need a comment in the code explaining this :)